### DISCUSSION TITLE
docker shows nothing when using mapped conf.yml

### DISCUSSION THREAD

**Discussion by becks0815** [Lissy93/dashy#1034] [Help and Support] [answered]
Hi,
I have a strange issue. When running dashy in docker using the conf file in the container using the following command

```
  docker run -d \
  -p 4000:80 \
  --name dashy1 \
  --restart=always \
  lissy93/dashy:latest
```
everything is fine. I can connect to my dashboard on port 4000 and the content of conf.yml is used/displayed. As soon as I try to use a mapped file instead:

```
  docker run -d \
  -p 4000:80 \
  -v /root/dashy_conf_orig.yml:/app/public/conf.yml \
  --name dashy2 \
  --restart=always \
  lissy93/dashy:latest
```

all I get is a screen with a dark blue screen which quickly shows the text "Dashy" if I hit cntrl+F5, and that's it. What I have tried so far without success:

- exported the default conf.yml via clipboard/local download and used it as config file
- changed the permissions of the config to 777
- launched a shell in the container and checked if the content of /app/public/conf.yml contains the same data like the one on the host system. Changed a character in the file, saved it to see if the change is also visible within the container (which was the case)
- Checked the text file with ultraedit to confirm it uses unix encoding and not dos by accident.
- Ran config through an YAML validator.

I think I am slowly running out of ideas what I could do next.

**Comment by becks0815** [comment, marked as answer]
Can be closed and deleted if necessary. I found the issue, it was a single leading space which maded it into the first line, so 

`pageInfo:`

became

` pageInfo:`

**Comment by InsaneHomer** [comment]
This drove me nuts for the last hour or so trying to restore a config.

The leading space is added to the downloaded dashy_conf.yml when downloaded via the GUI from
"Configuration Options" -> "View / Export Config" -> "Download as File"

the yml file is otherwise good. However that leading space leads to a blank page. Removing the leading space fixed the file.
